components/molecules/FeaturedHighlightsPanel/featured-highlights-panel.tsx

```diff
@@ -0,0 +1,26 @@
+import Link from "next/link";
+import React from "react";
+
+interface FeaturedHighlightsPanelProps {
+  highlightIds: { id: number; title: string }[];
+}
+const FeaturedHighlightsPanel = ({ highlightIds }: FeaturedHighlightsPanelProps) => {
+  return (
+    <div className="border max-w-[299px] rounded-lg p-5 space-y-4">
+      <h2 className="pb-2 text-lg border-b">Featured Highlights</h2>
+      <div className="flex flex-col gap-4 text-sm">
+        {highlightIds.map(({ id, title }) => (
+          <Link
+            className="transition hover:text-light-orange-10"
+            key={id}
+            href={`https://insights.opensauced.pizza/feed/${id}`}
```

Comment:
This link might want to take in account "localhost" or "deply-preview*"

Perhaps we can use the BASE_URL or a similar structure here.